griptape/tools/base_tool.py

```diff
@@ -187,3 +187,17 @@ def find_input_memory(self, memory_name: str) -> Optional[TaskMemory]:
             return next((m for m in self.input_memory if m.name == memory_name), None)
         else:
             return None
+
+    def to_native_tool_name(self, activity: Callable) -> str:
```

Comment:
I still don't know if I love the terminology "Native Tool". Open to suggestions on improving this before releasing.

Reply:
After thinking about it some more, "supported natively" is a common term, so we can probably leave it as "native tools."